doc/src/conditional-compilation.md

```diff
@@ -1,13 +1,21 @@
 # Conditional compilation
 
-LALRPOP support conditional compilation of public non-terminal declarations via
+LALRPOP support conditional compilation of non-terminal declarations via
 `#[cfg(feature = "FEATURE")]` attributes. If run in a build script LALRPOP will
 automatically pickup the features from `cargo` and use those. Alternatively an
 explicit set of features can be set using the `Configuration` type.
 
+Like rust's `cfg` attribute, the syntax accepts `not()`, `any()` and `all()`
```

Comment:
From the grammar it looks like this pr might also support nestings like `not(any(feature = "x"))`. Is this true?

Reply:
If so, lets so this off!